packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -72,113 +72,285 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
+///                   // unset, so that the NestedScrollView can control this
+///                   // inner scroll view.
+///                   // If the "controller" property is set, then this scroll
+///                   // view will not be associated with the NestedScrollView.
+///                   // The PageStorageKey should be unique to this ScrollView;
+///                   // it allows the list to remember its scroll position when
+///                   // the tab view is not on the screen.
+///                   key: PageStorageKey<String>(name),
+///                   slivers: <Widget>[
+///                     SliverOverlapInjector(
+///                       // This is the flip side of the SliverOverlapAbsorber
+///                       // above.
+///                       handle: NestedScrollView.sliverOverlapAbsorberHandleFor(context),
+///                     ),
+///                     SliverPadding(
+///                       padding: const EdgeInsets.all(8.0),
+///                       // In this example, the inner scroll view has
+///                       // fixed-height list items, hence the use of
+///                       // SliverFixedExtentList. However, one could use any
+///                       // sliver widget here, e.g. SliverList or SliverGrid.
+///                       sliver: SliverFixedExtentList(
+///                         // The items in this example are fixed to 48 pixels
+///                         // high. This matches the Material Design spec for
+///                         // ListTile widgets.
+///                         itemExtent: 48.0,
+///                         delegate: SliverChildBuilderDelegate(
+///                           (BuildContext context, int index) {
+///                             // This builder is called for each child.
+///                             // In this example, we just number each list item.
+///                             return ListTile(
+///                               title: Text('Item $index'),
+///                             );
+///                           },
+///                           // The childCount of the SliverChildBuilderDelegate
+///                           // specifies how many children this inner list
+///                           // has. In this example, each tab has a list of
+///                           // exactly 30 items, but this is arbitrary.
+///                           childCount: 30,
+///                         ),
 ///                       ),
 ///                     ),
-///                   ),
-///                 ],
-///               );
-///             },
-///           ),
-///         );
-///       }).toList(),
+///                   ],
+///                 );
+///               },
+///             ),
+///           );
+///         }).toList(),
+///       ),
 ///     ),
-///   ),
-/// )
+///   );
+/// }
 /// ```
 /// {@end-tool}
+///
+/// ## More on using [SliverAppBar]s with [NestedScrollView]s
```

Comment:
nit: maybe remove the "more on using" since its just fluff. "[SliverAppBar]s with [NestedScrollView]s" seems to work as a heading.